examples/configs/joint-core-bert.cfg

```diff
@@ -86,9 +86,10 @@ min_action_freq = 1
 # to a batch of Doc objects, which are preprocessed into Span objects to support
 # longer documents.
 [components.transformer.model]
-@architectures = "spacy-transformers.TransformerModel.v1"
+@architectures = "spacy-transformers.TransformerModel.v2"
 name = "roberta-base"
 tokenizer_config = {"use_fast": true}
+transformer_config = {"output_attentions": false}
```

Comment:
I think this should just be empty by default.

Reply:
As this is an example config, let's leave it here as such